### PR TITLE
fixed export for tago run new fields

### PR DESCRIPTION
**Pull request by vitorfdl**
Add new commands and fix the export for access policies on analysis.

Added:
- command copy-tab to duplicate tab in a dashboard
- command device-copy to copy the bucket from device to another

Fixed:
- Exporting access rules not replacing the analysis IDs
- Exporting Run Buttons not exporting Dynamic Values settings

### CODE REVIEW

**Review by mateuscardosodeveloper — APPROVED** [review]

File: src/prompt/pick-dashboard-id-from-tagoio.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import prompts from "prompts";
+
+import { Account } from "@tago-io/sdk";
+
+async function pickDashboardIDFromTagoIO(account: Account, message: string = "Which dashboard you want to pick?") {
```
Comment (Freddyminu) [inline src/prompt/pick-dashboard-id-from-tagoio.ts:5]: Which dashboard you want to pick? -> Which dashboard you want to **choose**?

File: src/commands/devices/copy-data.ts [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  const config = getEnvironmentConfig(options.environment);
+  if (!config || !config.profileToken) {
+    errorHandler("Environment not found");
+    return;
+  }
+
+  if (!options.from || !options.to) {
+    const account = new Account({ token: config.profileToken, region: "usa-1" });
+    options.from = await pickDeviceIDFromTagoIO(account, "Pick a device to copy the data from:");
+    options.to = await pickDeviceIDFromTagoIO(account, "Pick a device to copy the data to: ");
```
Comment (Freddyminu) [inline src/commands/devices/copy-data.ts:43]: Pick -> Choose

File: src/commands/devices/copy-data.ts [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+  for await (let data of dataStream) {
+    data = data.filter((x) => x.variable !== "payload");
+    total += data.length;
+    await deviceTo.sendData(data);
+    if (total >= amount) {
+      break;
+    }
+  }
+
+  successMSG(`> Data copy done. A total of ${total} registers were copied.`);
```
Comment (Freddyminu) [inline src/commands/devices/copy-data.ts:30]: `> Data transfer completed. A total of ${total} registers were copied.`

File: src/commands/dashboard/copy-tab.ts [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+  if (!dashID) {
+    dashID = await pickDashboardIDFromTagoIO(account);
+  }
+
+  const dashInfo = await account.dashboards.info(dashID);
+
+  if (!options.from || !options.to) {
+    let tabList = (dashInfo.tabs as DashboardTabs[]).map((x, i) => ({ title: `${i}. ${x.value} [${x.key}]`, value: x.key }));
+    if (!options.from) {
+      options.from = await pickTabFromDashboard(tabList, "Pick a tab to copy the data from:");
```
Comment (Freddyminu) [inline src/commands/dashboard/copy-tab.ts:118]: "Source Tab"

File: src/commands/dashboard/copy-tab.ts [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+    }
+
+    tabList = tabList.filter((x) => x.value !== options.from);
+    if (!options.to) {
+      options.to = await pickTabFromDashboard(tabList, "Pick a tab to copy the data to: ");
```
Comment (Freddyminu) [inline src/commands/dashboard/copy-tab.ts:123]: "Target Tab:"

**Review by Freddyminu — APPROVED** [review]

**Pull request merged**